pkg/services/ngalert/notifier/status.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // TODO: We no longer do apimodels at this layer, move it to the API.
 func (am *alertmanager) GetStatus() apimodels.GettableStatus {
-	config := &apimodels.PostableUserConfig{}
+	config := &apimodels.GettableUserConfig{}
```

Comment:
@JacobsonMT I know you changed this last week. Should I do an explicit conversion here? As far as I can tell, these structs are equivalent with the exception of `TemplateFileProvenances` in `GettableUserConfig`.